data/presets/man_made/ski_jump.json

```
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-ski_jumping",
+    "fields": [],
```

Perhaps these two fields would make sense?

```suggestion
    "fields": [
        `name`,
        `operator`
    ],
```

---

I wonder whether the structure itself will actually ever have a name of it's own… 😅 But probably better those two fields than nothing. Committing.